Author: Joerg Friedrich <Joerg.Friedrich@friedrich-kn.de>
Description:
 Adding support for all flags from magic.h in python-magic (Closes: #529354).

diff -Naurp file.orig/python/py_magic.c file/python/py_magic.c
--- file.orig/python/py_magic.c	2009-12-05 17:30:09.000000000 +0100
+++ file/python/py_magic.c	2009-12-05 18:35:22.000000000 +0100
@@ -303,11 +303,24 @@ static struct const_vals {
     { "MAGIC_SYMLINK", MAGIC_SYMLINK },
     { "MAGIC_COMPRESS", MAGIC_COMPRESS },
     { "MAGIC_DEVICES", MAGIC_DEVICES },
-    { "MAGIC_MIME", MAGIC_MIME },
+    { "MAGIC_MIME_TYPE", MAGIC_MIME_TYPE },
     { "MAGIC_CONTINUE", MAGIC_CONTINUE },
     { "MAGIC_CHECK", MAGIC_CHECK },
     { "MAGIC_PRESERVE_ATIME", MAGIC_PRESERVE_ATIME },
+    { "MAGIC_RAW", MAGIC_RAW},
     { "MAGIC_ERROR", MAGIC_ERROR},
+    { "MAGIC_MIME_ENCODING", MAGIC_MIME_ENCODING },
+    { "MAGIC_MIME", MAGIC_MIME },
+    { "MAGIC_APPLE", MAGIC_APPLE },
+    { "MAGIC_NO_CHECK_COMPRESS", MAGIC_NO_CHECK_COMPRESS },
+    { "MAGIC_NO_CHECK_TAR", MAGIC_NO_CHECK_TAR },
+    { "MAGIC_NO_CHECK_SOFT", MAGIC_NO_CHECK_SOFT },
+    { "MAGIC_NO_CHECK_APPTYPE", MAGIC_NO_CHECK_APPTYPE },
+    { "MAGIC_NO_CHECK_ELF", MAGIC_NO_CHECK_ELF },
+    { "MAGIC_NO_CHECK_TEXT", MAGIC_NO_CHECK_TEXT },
+    { "MAGIC_NO_CHECK_CDF", MAGIC_NO_CHECK_CDF },
+    { "MAGIC_NO_CHECK_TOKENS", MAGIC_NO_CHECK_TOKENS },
+    { "MAGIC_NO_CHECK_ENCODING", MAGIC_NO_CHECK_ENCODING },
     { NULL }
 };
 
